src/components/Day.jsx

```diff
@@ -4,151 +4,25 @@
-  events: [],
+  viewDate: PropTypes.instanceOf(Date),
 }
 
 class Day extends React.Component {
 
-  /**
-   * Compare function - compares by 'startsAs' member variable
-   * @param lhs Left hand side
-   * @param rhs Right hand side
-   * @returns {number} Comparison result
-   */
-  cmpByStart (lhs, rhs) {
-
-    if (lhs.startsAt < rhs.startsAt) {
-      return -1
-    } else if (lhs.startsAt > rhs.startsAt) {
-      return 1
-    } else {
-      return 0
-    }
-  }
-
-  /**
-   * Finds all overlayed events and returns updated events array with appear property
-   * @param props Props
-   * @returns {*} Updated events
-   */
-  findOverlayedEvents (props) {
-
-    var overlayed = []
-    var lastend = new Moment(0)
-    var events = props.events.sort(this.cmpByStart)
-
-    // Compares this event's start with the last end. If the start is after the last end,
-    // set appropriate appearances for all events in queue.
-    for (var evid in events) {
-
-      var start = new Moment(events[evid].startsAt)
-
-      // Compare
-      if (start.isAfter(lastend) || start.isSame(lastend)) {
-        let appearance = appearanceClass(overlayed.length)
-        events = applyAppearance(events, overlayed, appearance)
-        overlayed = []
-      }
-
-      // Queue the event
-      overlayed.push(evid)
-
-      // Set event's end as last end
-      if (new Moment(events[evid].endsAt).isAfter(lastend)) {
-        lastend = new Moment(events[evid].endsAt)
-      }
-    }
-    // FIXME: DUPLICATION!
-    // Set appearance for the last events
-    let appearance = appearanceClass(overlayed.length)
-    return applyAppearance(events, overlayed, appearance)
-  }
-
-  renderEvent (event) {
-    if (!this.props.displayFilter[event.type]) {
-      event.appear = 'hide'
-    }
-    const shown = (event.id == this.props.showDetailOn)
-
-    return (
-      <EventBox
-        key={event.id}
-        data={event}
-        detailShown={shown}
-        onClick={this.props.onDetailShow}
-        openFromBottom={this.props.id >= 3}
-        colored={this.props.colored}
-        onViewChange={this.props.onViewChange}
-        onDateChange={this.props.onDateChange}
-        linkNames={this.props.linkNames}
-        layout={this.props.layout}
-        screenSize={this.props.screenSize}
-      />
-    )
-
-  }
-
   render () {
+    const thisDay = moment(this.props.viewDate).isoWeekday(parseInt(this.props.id, 10) + 1)
+    const isToday = moment(thisDay).isSame(moment(), 'days')
+    const isSelected = moment(thisDay).isSame(moment(this.props.viewDate), 'days')
+    const className = `day ${isToday ? 'active' : ''} ${isSelected ? 'selected' : ''}`
 
-    const events = this.findOverlayedEvents(this.props)
-    const className = `day ${this.props.active ? 'active' : ''} ${this.props.selected ? 'selected' : ''}`
-    const weekDay = new Moment().isoWeekday(parseInt(this.props.id, 10) + 1).format('dddd')
     let dayTitle = ''
-
     if (this.props.active) {
```

Comment:
`weekDay` is not defined, but I don't see any point in setting title for an active day. Do also remove `title={dayTitle}` in JSX.